### ISSUE TITLE
Missing semicolon not linting in css

### ISSUE THREAD

**Issue by Srcky** [CSSLint/csslint#617]
When there is only one property for a certain selector, and it's missing a semicolon, CSSLint won't show it as error. For instance:
```css
a {
color: #0083e8
}
```
This won't show as error.

**Comment by mirceapiturca** [comment]
According to the W3C spec the parser will see a block containing a list of declarations. After parsing "color: #0083e8" and adding it to the declarations list the next token is an EOF (actually WHITESPACE and EOF,but WHITESPACE is ignored), this is perfectly valid.

Unless there is some specific rule set in CSSLint that will raise an error for missing semicolons, this is perfectly valid from a "grammar" point of view.

**Comment by XhmikosR** [comment]
Yeah, this isn't invalid CSS.
